docs/docs/components.mdx

```diff
@@ -552,12 +552,15 @@ Note: The `feature-dimension` for sequence and sentence features does not have t
   Include a [Tokenizer](./components.mdx#tokenizers) component before this component.
 
   You should specify what language model to load via the parameter `model_name`. See the below table for the
-  available language models.
-  Additionally, you can also specify the architecture variation of the chosen language model by specifying the
-  parameter `model_weights`.
-  The full list of supported architectures can be found in the
-  [HuggingFace documentation](https://huggingface.co/transformers/pretrained_models.html).
-  If left empty, it uses the default model architecture that original Transformers library loads (see table below).
+  currently supported language models. The weights to be loaded can be specified by the additional parameter
+  `model_weights`. If left empty, it uses the default model weights listed in the table.
+
+  Apart from the default pretrained model weights, further models can be used from
+  [HuggingFace models](https://huggingface.co/models) provided the following conditions are met:
+
+  * The model architecture is one of the supported language models
+  * The model has pretrained Tensorflow weights (check for `tf_model.h5` in the "Files and versions" section)
+  * The model uses the default tokenizer (e.g. `TFBertModel` must use `BertTokenizer`)
 
   ```
   +----------------+--------------+-------------------------+
```

Comment:
In the example below it might be nice to mention that the base configuration of `rasa/LaBSE` is a multi-language model. I'm mainly mentioning this because I think a major use-case for this component is related to language support and explaining that this model may already cover your language could be useful. 

We also have an algorithm whiteboard video on that model that we might be able to link to: https://www.youtube.com/watch?v=7tAWk_Coj-s/.

Reply:
Would a note like this work?

The `LaBSE` weights that are loaded as default for the `bert` architecture provide a multi-lingual model trained on 
  112 languages (see the original paper [here](https://arxiv.org/pdf/2007.01852.pdf)). We strongly encourage using this 
  as a strong baseline and testing your bot end-to-end before trying to optimize this component with other 
  weights/architectures.